docs/features/project/run.md

```diff
@@ -113,27 +127,46 @@ Executing `algokit project run hello` from the root of the workspace will concur
 
 Executing `algokit project run hello` from the root of `project_(a|b)` will execute `echo hello` in the `project_(a|b)` directory.
 
-### Controlling order of execution
+### Controlling Execution Order
 
-To control order of execution, simply define the order for a particular command as follows:
+Customize command execution in workspaces for precise control:
```

Comment:
I think we split the order for order from the concurrency docs. They are different things.

Reply:
@neilcampbell pushed some changes

Reply:
@aorumbayev I can't see the changes. Did you push them.

Reply:
@neilcampbell should be visible now